subsys/spm/Kconfig

```diff
@@ -71,6 +71,20 @@ config SPM_SERVICE_READ
 
 endif
 
+config SPM_ALLOW_NON_SECURE_RESET
+	bool "Allow Non-Secure reset"
+#FIXME: Change this to Default to `n` when the issue regarding doing a software
```

Comment:
You should remove the FixMe comment (perhaps add a minor inline Fixme in the default y sentence, pointing to a Jira/Github issue

Reply:
Moved.